zfit/util/exception.py

```diff
@@ -1,4 +1,4 @@
-#  Copyright (c) 2019 zfit
+#  Copyright (c) 2020 zfit
```

Comment:
Your library uses a lot of extensions. Last time I checked there were even some that are unused. Maybe we should think about reducing them?

Reply:
You mean `exceptions` I assume?

Yeah, it could be reduced maybe. But rather too many, too specific. They are often just to be more informative and with time we figure out better which are needed. But I agree, some unused could be removed